### DISCUSSION TITLE
Failed to create label

### DISCUSSION THREAD

**Discussion by jundonglin** [PMEAL/OpenPNM#2875] [Q&A] [answered]
I created a label following the tutorial, but printed it with keyerror
![屏幕截图 2023-12-12 133020](https://github.com/PMEAL/OpenPNM/assets/151015751/3a545178-349c-4475-8fcd-d5e4d63e708a)

![屏幕截图 2023-12-12 130205](https://github.com/PMEAL/OpenPNM/assets/151015751/58491ce1-e6ec-4f04-a776-6990d8d78a60)

I would appreciate it if anyone could help

**Comment by jgostick** [comment, marked as answer]
You have a typo in your code.  There is a space after the a.

**Reply by jundonglin** [reply]
Thank you very much. I overlooked the space
